docs/csharp/tutorials/inheritance.md

```diff
@@ -0,0 +1,294 @@
+
+  The second constructor uses the [base](../language-reference/keywords/base.md) keyword to pass the title and publisher name to the base class constructor. A derived class constructor should always call a constructor of its base class. If you don't make an explicit call to a base class constructor in your source code, the C# compiler automatically supplies a call to the base class' default or parameterless constructor.
+
+- A read-only `ISBN` property, which returns the `Book` object's International Standard Book Number, a unique 10- or 13-digit number. The ISBN is supplied as an argument to one of the `Book` constructors and is stored in the private `id` field.
+
+- A read-only `Author` property. The author name is supplied as an argument to both `Book` constructors and is stored in the private `authorName` field.
+
+- Two read-only price-related properties, `Price` and `Currency`. Their values are provided as arguments in a `SetPrice` method call. The price is stored in a private field, `bookPrice`. The `Currency` property is the three-digit ISO currency symbol (for example, USD for the U.S. dollar) and is stored in the private `ISOCurrencySymbol` field. ISO currency symbols can be retrieved from the @System.Globalization.RegionInfo.ISOCurrencySymbol property.
+
+- A `SetPrice` method, which sets the values of the `bookPrice` and `ISOCurrencySymbol` fields. These are the values returned by the `Price` and `Currency` properties.
+
+- Overrides to the `ToString` method (inherited from `Publication`) and the @System.Object.Equals(System.Object) and @System.Object.GetHashCode methods (inherited from @System.Object).
+
+  Unless it is overridden, the @System.Object.Equals(System.Object) method tests for reference equality. That is, two object variables are considered to be equal if they refer to the same object. In the case of the `Book` class, on the other hand, two `Book` objects should be equal if they have the same ISBN.
+
+  When you override the @System.Object.Equals(System.Object) method, you must also override the @System.Object.GetHashCode method, which returns a value that the runtime uses to store items in hashed collections for efficient retrieval. The hash code should return a value that's consistent with the test for equality. Since we've overridden @System.Object.Equals(System.Object) to return `true` if the ISBN properties of two `Book` objects are equal, we return the hash code computed by calling the @System.String.GetHashCode method of the string returned by the `ISBN` property.
+
+The following figure illustrates the relationship between the `Book` class and `Publication`, its base class.
+
+![Publication and Book classes](media/book-class.jpg)
+
+We can now instantiate a `Book` object, invoke both its unique and inherited members, and pass it as an argument to a method that expects either a parameter of the `Publication` or a parameter of type `Book`, as the following example shows.
```

Comment:
I think there's a missing word. did you mean "... of the `Publication` __type__ or a parameter..."